pkgs/sops-install-secrets/main.go

```diff
@@ -30,7 +30,9 @@ type secret struct {
 	Key          string     `json:"key"`
 	Path         string     `json:"path"`
 	Owner        string     `json:"owner"`
+	UID          int        `json:"uid"`
```

Comment:
Is it possible to check for null || int here?

Reply:
Changed to pointers